src/services/utils.js

```diff
@@ -196,7 +196,7 @@ export const fetchDefaultParametersValues = () => {
     });
 };
 export const getQueryParamsList = (params, paramName) => {
-    if (params !== undefined && params.length > 0) {
+    if (params != null && Array.isArray(params) && params.length > 0) {
```

Comment:
(params != null && Array.isArray(params)) || params !== undefined

Reply:
`params !== null ` is equivalent to `params != null  && params != undefined`